compliance_checker/cf/cf.py

```diff
@@ -723,9 +725,9 @@ def check_ancillary_data(self, ds):
             anc_result = Result(BaseCheck.HIGH, name=('§3.4 Ancillary Variables', k))
             msgs = []
 
-            if not isinstance(anc, str):
+            if not isinstance(anc, str) and not isinstance(anc, unicode):
```

Comment:
This is fairly common (without the use of `six`) to support both py2 and py3:

```
try:
    basestring
except NameError:
    basestring = str
```

Then just always do `isinstance(foo, basestring)`